src/Altinn.Platform/Altinn.Platform.Events/Events/Startup.cs

```diff
@@ -68,9 +68,16 @@ public void ConfigureServices(IServiceCollection services)
 
             _logger.LogInformation("Startup // ConfigureServices");
 
-            services.AddControllersWithViews();
+            services.AddControllers().AddJsonOptions(options =>
+           {
+               options.JsonSerializerOptions.IgnoreNullValues = true;
+               options.JsonSerializerOptions.PropertyNamingPolicy = System.Text.Json.JsonNamingPolicy.CamelCase;
+           });
+
             services.AddHealthChecks().AddCheck<HealthCheck>("events_health_check");
+            services.Configure<AzureCosmosSettings>(Configuration.GetSection("AzureCosmosSettings"));
```

Comment:
A fancy pancy change. 
```suggestion
            services.Configure<AzureCosmosSettings>(Configuration.GetSection(nameof(AzureCosmosSettings)));
```